src/core/ext/transport/binder/wire_format/wire_reader_impl.cc

```diff
@@ -109,11 +107,12 @@ std::shared_ptr<WireWriter> WireReaderImpl::SetupTransport(
 
 void WireReaderImpl::SendSetupTransport(Binder* binder) {
   binder->Initialize();
```

Comment:
binder->PrepareTransaction() will not be executed if it is a part of the VLOG(2) statement. Hence moving it out of the VLOG statement. Same for other calls below.